data/students.json

```diff
@@ -1,75 +1,314 @@
 {
   "students": [
     {
-      "displayName": "Guillory Martin",
+      "displayName": "Trinidad Escobar",
+      "slug": "escobar",
+      "year": 2015,
```

Comment:
I would recommend changing all the years to strings (as they were originally were).  Less potential for bugs, I suspect.

Reply:
👍🏻